src/Spout/Writer/CSV/Writer.php

```diff
@@ -56,14 +59,29 @@ public function setFieldEnclosure($fieldEnclosure)
     }
 
     /**
+     * Set if a BOM has to be added to the file
+     *
+     * @param bool $addBomToFile
+     *
+     * @return Writer
+     */
+    public function addBomToFile($addBomToFile)
```

Comment:
rename `setShouldAddBOM($shouldAddBOM)` to be consistent with the rest of the codebase